Formula/geant4.rb

```diff
@@ -4,6 +4,7 @@ class Geant4 < Formula
   url "https://geant4-data.web.cern.ch/geant4-data/releases/source/geant4.10.07.tar.gz"
   version "10.7.0"
   sha256 "776ea45230d26fffebf0cf7a342af5131135759a0f70e1b4a1a401f1d1eaad4a"
+  license :cannot_represent
```

Comment:
I think this is fine, but can you add a comment with a link to the license just before it?

Reply:
Done, and I've also made sure the license file is installed (I'll patch that upstream as well)